profile/README.md

```diff
@@ -0,0 +1,65 @@
+<p align="center">
+  <img src="https://security-summer-school.github.io/SSS-logo.png" width="215" alt="Security Summer School" />
+</p>
+<p align="center">
+  <img src="https://img.shields.io/github/contributors/security-summer-school/binary" />
+    <a href="https://bit.ly/DiscordSecuritySummerSchool">
+    <img src="https://img.shields.io/discord/715162845305634816.svg?label=discord&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2" />
+  </a>
+</p>
+
+### Security Summer School is a security workshop held by people passionate about security, professors, PhD, MSci, BSc students from University POLITEHNICA of Bucharest and industry partners Ixia, a Keysight Business.
+
+
+## Projects
+
+### :bulb: Learn the most known vulnerabilities
+
+From zero-to-hero the four week event holds lessons and workshops two or three times a week on vulnerabilities and how to identify and get advantage from them.
+
+### :mag_right: Hands-on learning
+
+At the end of each presentation, you will have to solve problems, practicing the presented vulnerabilities.
+These will run as a CTF where you should find the flag and demonstrate your power.
+
+### :trophy: CTF & Prizes!
+
+Two CTFs will be organized, one in the middle of the summer school and the other at the end of it.
+Those who get involved will also receive prizes.
```

Comment:
Remove this. We don't have winner prizes anymore. We're in communism, since @teodutu 's involvement, and everyone gets something.